python/lsst/ts/observatory/control/script_queue.py

```diff
@@ -341,12 +341,18 @@ async def get_script_schema(self, is_standard: bool, script: str) -> str:
         await self.queue_remote.cmd_showSchema.set_start(
             isStandard=is_standard,
             path=script,
-            timeout=self.fast_timeout,
+            timeout=self.long_timeout,
```

Comment:
This parameter is completely ignored in the code in this function. Can you remove it or make sure that it is used?

Reply:
Not sure I understand what you mean... are you suggesting the timeout is ignored?

Reply:
Yes, that's what I was suggesting but I see now that that is not the case. Sorry, my bad.